build/docker/debian10/amd64/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Dockerfile for building DO client components for Debian 10 amd64
+# First, install the docker extension for VSCode. Then you can right-click on this file
+# and choose Build Image. Give it a name and it will build the image.
+#
+# Open interactive terminal into the image in a container:
+# docker run -ti --rm --entrypoint=/bin/bash -v <project root dir>:/code -v <build root dir>:/build <image_name>
+# Example:
+# docker run -ti --rm --entrypoint=/bin/bash -v D:\do-client-lite:/code -v D:\temp\build_client_lite\arm-linux-debug:/build custom-debian10-amd64
+
+FROM amd64/debian:buster
```

Comment:
>amd64/debian [](http://example.com/codeflow?start=5&length=12)

We should use the golden images as the base. Without any prefix, it defaults to dockerhub.